spec/unit/opds_spec.lua

```diff
@@ -6,53 +7,54 @@ DON'T USE THIS PAGE FOR SCRAPING.
 
 Seriously. You'll only get your IP blocked.
 
-Download http://www.gutenberg.org/feeds/catalog.rdf.bz2 instead,
+Download https://www.gutenberg.org/feeds/catalog.rdf.bz2 instead,
 which contains *all* Project Gutenberg metadata in one RDF/XML file.
 
---><feed xmlns="http://www.w3.org/2005/Atom" xmlns:opds="http://opds-spec.org/2010/catalog" xmlns:dcterms="http://purl.org/dc/terms/" xmlns:opensearch="http://a9.com/-/spec/opensearch/1.1/" xmlns:relevance="http://a9.com/-/opensearch/extensions/relevance/1.0/" xml:base="http://m.gutenberg.org/ebooks.opds/?format=opds">
-<id>http://m.gutenberg.org/ebooks.opds/</id>
-<updated>2014-05-17T12:04:49Z</updated>
+--><feed xmlns="http://www.w3.org/2005/Atom" xmlns:opds="http://opds-spec.org/2010/catalog" xmlns:dcterms="http://purl.org/dc/terms/" xmlns:opensearch="http://a9.com/-/spec/opensearch/1.1/" xmlns:relevance="http://a9.com/-/opensearch/extensions/relevance/1.0/">
+<id>http://www.gutenberg.org/ebooks.opds/</id>
```

Comment:
You might want to tell them about this `<id>` if they're planning to switch completely to https. ;-)